doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -985,15 +984,27 @@ Generation of induction principles with ``Scheme``
   induction for objects in type :n:`@reference`.
 
   :n:`@ident`
-    The name of the scheme. If not provided, the scheme name will be determined automatically
-    from the sorts involved.
+    The name of the scheme. If not provided, the scheme name will be determined
+    automatically from the sorts involved.
+
+  :n:`Induction fo @reference Sort @sort_family`
+    Defines a dependent elimination principle for a family of `@sort_family`s.
 
   :n:`Minimality for @reference Sort @sort_family`
-    Defines a non-dependent elimination principle more natural for inductively defined relations.
+    Defines a non-dependent elimination principle for a constant family of `@sort_family`.
+
+  :n:`Elimination for @reference Sort @sort_family`
+    Defines a dependent case-analysis for a family of `@sort_family`s.
```

Comment:
```suggestion
    Defines a dependent case analysis for a family of `@sort_family`s.
```

Why "case analysis" and not "induction principle" or "elimination principle"?  Is it a distinct concept or just a wording variation?

Reply:
Same concept, I've cleared up the wording now.